Makefile

```diff
@@ -52,7 +52,7 @@ build-linux-amd64: ## Create the kubicorn executable for Linux 64-bit OS in the
 	-w /go/src/github.com/kubicorn/kubicorn \
 	-v ${PWD}:/go/src/github.com/kubicorn/kubicorn \
 	-e GOPATH=/go \
-	--rm golang:1.8.1 make docker-build-linux-amd64
+	--rm golang:1.9.3 make docker-build-linux-amd64
```

Comment:
Should this possible reference the SHELL_IMAGE variable instead of hardcoding it here?

Reply:
sounds good, i'll change it